content/en/blog/_posts/2024-07-22-feature-gates_client-go.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: blog
+title: 'Introducing Feature Gates to Client-Go: Enhancing Flexibility and Control'
+date: 2024-07-22
+slug: feature-gates-in-client-go
+author: >
+ Ben Luddy (Red Hat),
+ Lukasz Szaszkiewicz (Red Hat)
+---
+
+## Introduction
+
+Kubernetes contributors use on-off switches called “feature gates” to manage the risk of adding a new feature to Kubernetes components. 
```

Comment:
```suggestion
Kubernetes uses on-off switches called _feature gates_ to manage the risk of adding a new feature to Kubernetes and its components.
```

Reply:
The original uses "Kubernetes Components" as it's used in https://kubernetes.io/docs/concepts/overview/components/ to contrast binaries like kube-apiserver, kube-controller-manager, and kube-scheduler, from client-go, which is part of Kubernetes but is not a "Kubernetes Component". How about something like:

> Kubernetes components use on-off switches called _feature gates_ to manage the risk of adding a new feature.